.jenkins/Jenkinsfile

```diff
@@ -218,9 +218,25 @@ def win2016CrossCompile(String build_type, String has_quote_provider = 'OFF') {
 
                   bat """
                       vcvars64.bat x64 && \
-                      cmake.exe ${WORKSPACE} -G Ninja -DCMAKE_BUILD_TYPE=${build_type} -DBUILD_ENCLAVES=ON -DHAS_QUOTE_PROVIDER=${has_quote_provider} -DNUGET_PACKAGE_PATH=C:/oe_prereqs -Wdev && \
+                      cmake.exe ${WORKSPACE} -G Ninja -DCMAKE_BUILD_TYPE=${build_type} -DBUILD_ENCLAVES=ON -DHAS_QUOTE_PROVIDER=${has_quote_provider} -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCPACK_GENERATOR=NuGet -Wdev && \
                       ninja.exe && \
-                      ctest.exe -V -C ${build_type} --timeout ${CTEST_TIMEOUT_SECONDS}
+                      ctest.exe -V -C ${build_type} --timeout ${CTEST_TIMEOUT_SECONDS} && \
+                      echo current working directory is %cd% && \
+                      cpack.exe -D CPACK_NUGET_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY && \
```

Comment:
I think a packaging test would be separate from the cross-compile test.

I would recommend making a new function for testing the Windows packages as opposed to adding to existing ones.